src/java.base/share/classes/java/util/Scanner.java

```diff
@@ -78,7 +78,16 @@
  *     }
  * }
  *
- * <p>As another example, this code allows {@code long} types to be
+ * <p>To read lines from {@link System#in} with {@link System##stdin.encoding
```

Comment:
Other 2 example paragraphs start with "this code allows..." and I think we should start this paragraph with: (you might need to wrap lines)
```suggestion
 * <p>This code reads lines from {@link System#in} with {@link System##stdin.encoding
```

Reply:
Thanks. Modified